app/assets/stylesheets/application.scss

```diff
@@ -223,7 +223,7 @@ path[data-event="geek-girl-con"] {
 }
 
 .table-striped > tbody > tr:nth-of-type(odd) {
-    background-color: #eeeeee;
+    background-color: #eeeeee !important;
```

Comment:
😭

Reply:
Yeah... I had to add this to prevent the background color changing to the bootstrap default color when I changed the html of the table to remove the Return Game button and add the return time/date. It's weird.